projects/demo/src/modules/app/home/examples/main-standalone-optional.md

```diff
@@ -5,7 +5,6 @@ import {TuiRootModule, TuiDialogModule, TuiAlertModule, TuiPushModule} from '@ta
 bootstrapApplication(AppComponent, {
   providers: [
     provideAnimations(),
-    provideRouter(routes),
     importProvidersFrom(
       TuiRootModule,
       TuiDialogModule,
```

Comment:
In 4.0 we no longer need dialogs/alerts in here, there are no providers in them and they are not required in the main module anymore. I guess we need to remove this whole section about optional modules.

Reply:
fixed